### PR TITLE
#826 Blocking example updated to use Blocker

### PR DESCRIPTION
**Pull request by Slakah**
See: #826 for brief explanation.

### CODE REVIEW

File: site/src/main/mdoc/concurrency/basics.md [hunk 1]
```diff
@@ -114,28 +114,27 @@ Blocking a thread means that it is being wasted and nothing else can be schedule
...
   for {
-    _ <- contextShift.evalOn(blockingEC)(blockingOp) // executes on blockingEC
-    _ <- doSth()                                     // executes on contextShift
+    _ <- blocker.blockOn(blockingOp) // executes on blocker, backed by cached thread pool
+    _ <- doSth()                     // executes on contextShift
   } yield ()
+}
 ```
 
-For more convenient tools for this pattern, see [linebacker](https://github.com/ChristopherDavenport/linebacker).
```
Comment (Slakah) [inline site/src/main/mdoc/concurrency/basics.md:138]: Removed linebacker comment, as it feels like `Blocker[IO]` would replace linebacker for most applications.

@ChristopherDavenport does this make sense? I could add some references to blocking related things in a reference section if we want to keep the linebacker link.

File: site/src/main/mdoc/concurrency/basics.md [hunk 2]
```diff
@@ -114,28 +114,27 @@ Blocking a thread means that it is being wasted and nothing else can be schedule
 As mentioned, this can be very dangerous and it's best to use dedicated thread
 pool for blocking operations. This way they won't interfere with CPU-bound part of application.
 
-`cats.effect.IO` and `monix.eval.Task` provide `shift` operator which can switch computation to different thread pool.
```
Comment (Slakah) [inline site/src/main/mdoc/concurrency/basics.md:117]: It felt like `.shift` explanations shouldn't really be covered in the `Blocker` section. Was aiming for terseness with the changes made.

**Comment by Slakah** [comment]
Closes #826

**Comment by djspiewak** [comment]
Excellent changes; thank you!

**Pull request merged**